### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by hieunk58**

### CODE REVIEW

File: app/assets/javascripts/static_pages.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (nhunght) [inline app/assets/javascripts/static_pages.coffee]: remove this file

File: app/assets/stylesheets/static_pages.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (nhunght) [inline app/assets/stylesheets/static_pages.scss]: remove this file

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title(page_title)
```
Comment (nhunght) [inline app/helpers/application_helper.rb]: remove ()

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -1,2 +1,10 @@
...
+    base_title = "Food and Drink Website"
```
Comment (nhunght) [inline app/helpers/application_helper.rb]: i18n

File: app/helpers/static_pages_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```
Comment (nhunght) [inline app/helpers/static_pages_helper.rb]: remove this file

File: app/views/layouts/_footer.html.erb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<footer>
+  <small>
+    <p>Food and Drink Web designed by Framgia Team</p>
```
Comment (nhunght) [inline app/views/layouts/_footer.html.erb]: i18n

File: app/views/static_pages/help.html.erb [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<% provide(:title, "Help") %>
+<html>
+  <head>
+    <title><%= yield(:title) %> | Food and Drink Website</title>
+  </head>
+  <body>
+    <h1>Help</h1>
```
Comment (thangtx88) [inline app/views/static_pages/help.html.erb]: I18n

File: app/views/static_pages/help.html.erb [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+    <p>Coming soon!</p>
```
Comment (thangtx88) [inline app/views/static_pages/help.html.erb]: I18n

File: app/views/static_pages/home.html.erb [hunk 9]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= yield(:title) %> | Food and Drink Website</title>
```
Comment (thangtx88) [inline app/views/static_pages/home.html.erb]: I18n

File: Gemfile [hunk 10]
```diff
@@ -1,18 +1,25 @@
 source "https://rubygems.org"
 
 
+<<<<<<< b24f51fb4335cc9eb0da32447b786a4c6a5f802d
```
Comment (thangtx88) [inline Gemfile]: ???

File: Gemfile [hunk 11]
```diff
@@ -1,18 +1,25 @@
...
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem "rails", "4.2.6"
 # Use sqlite3 as the database for Active Record
 
+=======
```
Comment (thangtx88) [inline Gemfile]: ????

File: Gemfile [hunk 12]
```diff
@@ -1,18 +1,25 @@
...
+# Bundle edge Rails instead: gem "rails", github: "rails/rails"
+gem "rails", "4.2.6"
+
+gem "bootstrap-sass", "3.2.0.0"
+>>>>>>> create static pages
```
Comment (thangtx88) [inline Gemfile]: ??????

File: app/views/layouts/_footer.html.erb [hunk 13]
```diff
@@ -0,0 +1,12 @@
+<footer>
+  <small>
+    <p><%= t "views.footer_title" %></p>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "About Us" %></li>
```
Comment (thangtx88) [inline app/views/layouts/_footer.html.erb]: I18n

File: app/views/layouts/_footer.html.erb [hunk 14]
```diff
@@ -0,0 +1,12 @@
...
+      <li><%= link_to "Help" %></li>
```
Comment (thangtx88) [inline app/views/layouts/_footer.html.erb]: I18n

File: app/views/layouts/_footer.html.erb [hunk 15]
```diff
@@ -0,0 +1,12 @@
...
+      <li><%= link_to "Contact" %></li>
```
Comment (thangtx88) [inline app/views/layouts/_footer.html.erb]: I18n

File: app/views/layouts/_header.html.erb [hunk 16]
```diff
@@ -0,0 +1,16 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+    <div class="logo">
+      <%= link_to t("views.food"), '#', id: "logo" %>
+    </div>
+    <div class="main">
+      <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home",       '#' %></li>
```
Comment (thangtx88) [inline app/views/layouts/_header.html.erb]: I18n

File: app/views/layouts/_header.html.erb [hunk 17]
```diff
@@ -0,0 +1,16 @@
...
+        <li><%= link_to "Category",   '#' %></li>
```
Comment (thangtx88) [inline app/views/layouts/_header.html.erb]: I18n

File: app/views/layouts/_header.html.erb [hunk 18]
```diff
@@ -0,0 +1,16 @@
...
+        <li><%= link_to "Sign up",    '#' %></li>
```
Comment (thangtx88) [inline app/views/layouts/_header.html.erb]: I18n

File: app/views/layouts/_header.html.erb [hunk 19]
```diff
@@ -0,0 +1,16 @@
...
+        <li><%= link_to "Sign in",    '#' %></li>
```
Comment (thangtx88) [inline app/views/layouts/_header.html.erb]: I18n

File: app/views/layouts/_header.html.erb [hunk 20]
```diff
@@ -0,0 +1,16 @@
...
+      </ul>
+    </nav>
+    </div>
+  </header>
+
```
Comment (thangtx88) [inline app/views/layouts/_header.html.erb]: end file

File: app/views/layouts/application.html.erb [hunk 21]
```diff
@@ -1,14 +1,16 @@
...
+  <title><%= full_title(yield(:title)) %></title>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
 </head>
 <body>
-
-<%= yield %>
-
+  <%= render 'layouts/header' %>
```
Comment (thangtx88) [inline app/views/layouts/application.html.erb]: double quote

File: app/views/layouts/application.html.erb [hunk 22]
```diff
@@ -1,14 +1,16 @@
...
+  <div class="container">
+    <%= yield %>
+  </div>
+   <%= render 'layouts/footer' %>
```
Comment (thangtx88) [inline app/views/layouts/application.html.erb]: double quote

File: app/assets/stylesheets/application.css [hunk 23]
```diff
@@ -13,3 +13,4 @@
  *= require_tree .
  *= require_self
  */
+ @import url(http://fonts.googleapis.com/css?family=Roboto);
```
Comment (dieunb) [inline app/assets/stylesheets/application.css]: :+1:

File: app/views/layouts/application.html.erb [hunk 24]
```diff
@@ -1,14 +1,16 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>BaseApp</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <title><%= full_title(yield(:title)) %></title>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: remove the first  `()`

File: app/views/static_pages/help.html.erb [hunk 25]
```diff
@@ -0,0 +1,10 @@
+<% provide(:title, "Help") %>
```
Comment (dieunb) [inline app/views/static_pages/help.html.erb]: remove `()`

File: config/application.rb [hunk 26]
```diff
@@ -17,9 +17,8 @@ class Application < Rails::Application
     # config.time_zone = 'Central Time (US & Canada)'
 
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
-    # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
-    # config.i18n.default_locale = :de
-
+    config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
```
Comment (dieunb) [inline config/application.rb]: double quote

File: config/database.yml.example [hunk 27]
```diff
@@ -1,25 +0,0 @@
-# SQLite version 3.x
```
Comment (dieunb) [inline config/database.yml.example]: don't delete this file, pls rollback it

File: config/routes.rb [hunk 28]
```diff
@@ -1,56 +1,4 @@
...
-  #   resources :posts, concerns: :toggleable
-  #   resources :photos, concerns: :toggleable
-
-  # Example resource route within a namespace:
-  #   namespace :admin do
-  #     # Directs /admin/products/* to Admin::ProductsController
-  #     # (app/controllers/admin/products_controller.rb)
-  #     resources :products
-  #   end
+  root 'static_pages#home'
```
Comment (dieunb) [inline config/routes.rb]: double quote

**Pull request merged**